frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -230,8 +230,9 @@ export function FunnelInvalidExclusionState(): JSX.Element {
 
 const SAVED_INSIGHTS_COPY = {
     [`${SavedInsightsTabs.All}`]: {
-        title: 'There are no insights $CONDITION.',
-        description: 'Once you create an insight, it will show up here.',
+        title: 'No insights matched your search for $CONDITION.',
+        description:
+            'Refine your keyword search, or try using other filters such as type, last modified or created by.',
```

Comment:
I think we would need to be a bit more specific here. If there are no insights, we still want to show the old copy. Only if there is a search condition (and _some_ existing insights, do we want to show this new copy.